### PR TITLE
fix: re-enable reading reminder toggle

### PR DESCRIPTION
**Pull request by omBratteng**
## Changes

Re-adds the possibility to actually toggle reading reminder that got removed in #3151

### Preview domain
https://fix-reading-reminder-toggle.preview.app.daily.dev

### CODE REVIEW

File: packages/webapp/pages/account/notifications.tsx [hunk 1]
```diff
@@ -166,8 +175,6 @@ const AccountNotificationsPage = (): ReactElement => {
       NotificationCategory.Product,
     );
 
-    onToggleReadingReminder(!isSubscribed);
-
```
Comment (omBratteng) [inline packages/webapp/pages/account/notifications.tsx:170]: Removed it from onTogglePush, so it doesn't get automagically enabled on notification toggle.

**Review by rebelchris — CHANGES_REQUESTED** [review]
Oh somehow missed this indeed.
But now it will sub/unsub on the global push agian so we'll need to make some changes there so it doesn't mutate on those

**Review by rebelchris — APPROVED** [review]

File: packages/webapp/pages/account/notifications.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/webapp/pages/account/notifications.tsx:170]: Yeah somehow totally missed this one 👍

**Review by sshanzel — APPROVED** [review]
🚀

**Pull request merged**